plotHist.cc

```diff
@@ -57,7 +57,7 @@ void plotHistBin(int q2Bin, int effIndx, int parity)
   string confString = "plotHist_d/KDEhist_"+shortString;
 
   // Load variables and dataset
-  string filename = Form("effDataset_b%i.root",q2Bin);
+  string filename = Form("/eos/cms/store/user/fiorendi/p5prime/effKDE/%i/lmnr/effDataset_b%i_%i.root",year,q2Bin,year);
```

Comment:
Even if in the README file we do not include instructions on how to use the plot macros, I would add a caveat with the lines that need customisation before running

Reply:
addressed [here](https://github.com/CMSKStarMuMu/eff-KDE/pull/2/commits/0d1163fbc6b25f3bb7966a660170b1394346356d)